src/components/tabs/tabs.tsx

```diff
@@ -81,11 +82,21 @@ export const EuiTabs = forwardRef<EuiTabRef, PropsWithChildren<EuiTabsProps>>(
     const tabsStyles = euiTabsStyles(euiTheme);
     const computedStyles = [
       tabsStyles.euiTabs,
-      tabsStyles[`size_${size}`],
+      tabsStyles[size],
       bottomBorder && tabsStyles.bottomBorder,
-      expand && tabsStyles.expanded,
     ];
 
+    const tabItems = React.Children.map(children, (child) => {
+      if (React.isValidElement(child)) {
+        return cloneElementWithCss(child, {
+          // we're passing the parent `size` and `expand` down to the children
+          size: size,
+          expand: expand,
+          ...child.props,
+        });
+      }
+    });
```

Comment:
Basically, I'm copying some props from the parent to the children because it makes it easier to apply the styles.